src/main/java/com/connectCo/domain/event/controller/EventController.java

```diff
@@ -20,6 +24,50 @@ public class EventController {
 
     private final EventService eventService;
 
+
+    @Operation(summary = "이벤트 생성 API")
+    @PostMapping("")
+    public BaseResponse<EventIdResponse> createEvent (@RequestPart(value = "eventImages", required = false) List<MultipartFile> eventImages, @RequestBody EventCreateRequest request){
+        return BaseResponse.onSuccess(eventService.createEvent(eventImages, request));
+    }
```

Comment:
@RequestPart와 @RequestBody를 동시에 사용하는 것은 불가능하기 때문에 다음과 같이 둘 다 @RequestPart로 변경하는게 좋을 거 같아요!!! 또한 한 줄에 글자 수를 적게 하는 것이 코드 디자인에서 좋은 방법이에요!!!
```suggestion
    public BaseResponse<EventIdResponse> createEvent (
            @RequestPart(value = "eventImages", required = false) List<MultipartFile> eventImages, 
            @RequestPart EventCreateRequest request){
        return BaseResponse.onSuccess(eventService.createEvent(eventImages, request));
    }
```